invenio_pidstore/config.py

```diff
@@ -19,26 +19,16 @@
 
 """Pidstore config."""
 
-from __future__ import unicode_literals
+# TODO: This file is to be removed
 
-import pkg_resources
+from __future__ import unicode_literals
 
 PIDSTORE_PROVIDERS = [
+    'invenio_pidstore.providers.datacite:DataCite',
     'invenio_pidstore.providers.local_doi:LocalDOI',
     'invenio_records.providers.recid:RecordID',
 ]
 
-try:
```

Comment:
@kiryx please keep the code somewhere. We should not include DataCite if `datacite` package is not installed.